src/Sentry/Internal/Enricher.cs

```diff
@@ -33,10 +33,14 @@ public void Apply(IEventLike eventLike)
         // Operating System
         if (!eventLike.Contexts.ContainsKey(OperatingSystem.Type))
         {
-            // RuntimeInformation.OSDescription is throwing on Mono 5.12
-            if (!PlatformAbstractions.Runtime.Current.IsMono())
-            {
-                eventLike.Contexts.OperatingSystem.RawDescription = RuntimeInformation.OSDescription;
+            // RuntimeInformation.OSDescription is throwing on Mono 5.12 and .net 4.6.2 on Linux/macOS
+            try {
+                if (!PlatformAbstractions.Runtime.Current.IsMono())
```

Comment:
Is the check now throwing too?

Reply:
the check could be outside of try-catch, it's the call below that throws on net4.6.2 on Linux and macOS.

Reply:
anyway... let me check if it throws on netstandard too and I'll add an ifdef